iib/workers/tasks/build.py

```diff
@@ -916,6 +931,8 @@ def handle_add_request(
     :param str overwrite_from_index_token: the token used for overwriting the input
         ``from_index`` image. This is required for non-privileged users to use
         ``overwrite_from_index``. The format of the token must be in the format "user:password".
+    :param str distribution_scope: the scope for distribution of the index image, defaults to
+        'PROD'.
```

Comment:
This doesn't seem to default to `PROD` based on `distribution_scope=None,` in the function signature.

Reply:
Maybe I am being silly, but if the default of what the distribution_scope will be is prod even through the default for the parameter as none? or is that not the way to think about it?

Reply:
It's just that the way it was worded makes it sound like the `distribution_scope` keyword argument defaults to `'PROD'` even though it's code below that does. If you rephrase it, then it's useful to have this information.